### PR TITLE
refactor get, update and create API

### PR DESCRIPTION
**Pull request by aabarata**

### CODE REVIEW

File: .ciconfig [hunk 1]
```diff
@@ -11,7 +11,7 @@ pipeline {
           sh 'npm install'
         }
         dir('linkurious-server') {
-          git branch: 'develop', credentialsId: 'dd38fcea-d471-4856-a009-4029127c6450', url: 'git@github.com:Linkurious/linkurious-server.git'
+          git branch: 'query-ux', credentialsId: 'dd38fcea-d471-4856-a009-4029127c6450', url: 'git@github.com:Linkurious/linkurious-server.git'
```
Comment (maxime-allex) [inline .ciconfig]: Dangerous, this should not be touched

File: package.json [hunk 2]
```diff
@@ -82,6 +82,8 @@
     "pretty-quick": "^1.8.0"
   },
   "dependencies": {
-    "superagent": "^3.5.2"
+    "superagent": "^3.5.2",
+    "rxjs": "^6.3.3",
```
Comment (maxime-allex) [inline package.json]: remove rxjs from here and move it in shared from dev-dependencies to dependencies

File: .ciconfig (hunk 1 above)
Comment (aabarata) [reply .ciconfig]: Done!

File: package.json (hunk 2 above)
Comment (aabarata) [reply package.json]: Done!

**Pull request merged**